src/libs/sfdk/virtualmachine.h

```diff
@@ -126,6 +127,10 @@ class SFDK_EXPORT VirtualMachine : public QObject
     void setStorageSizeMb(int storageSizeMb, const QObject *context,
             const Functor<bool> &functor);
 
+    int freeStorageSizeMb() const;
```

Comment:
I would prefer ordering `freeStorageSizeMb` directly after `storageSizeMb` and keep these four stuck together without separating blank line. Same ordering in .cpp.